cms/templates/cms/home_page.html

```diff
@@ -118,79 +118,74 @@ <h5 class="upper-case">Save Money &amp; Time</h5>
+                  <a href="{{ program.programpage.url }}" class="program-link">
                 {% endif %}
-                    <div class="program-thumbnail-image-wrap">
-                      {% if program.programpage and program.programpage.thumbnail_image %}
-                        {% image program.programpage.thumbnail_image fill-690x530 as thumbnail_image %}
-                        <img src="{{ thumbnail_image.url }}" alt="course image for {{ program }}"
-                             >
-                      {% else %}
-                        <img src="{% static 'images/course-thumbnail.png' %}"
-                             alt="course image for {{ program }}" style="opacity: .4">
-                      {% endif %}
-                      <div class="program-bottom-info">
-                      <div class="program-title">{{ program.programpage.title }}</div>
-                      <div class="num-courses">
-                        <span class="meta-courses">
-                          {% blocktrans count counter=program.course_set.count %}
-                            {{ counter }} course
-                          {% plural %}
-                            {{ counter }} courses
-                          {% endblocktrans %}
-                        </span>
-                      </div>
-                      </div>
-                    </div>
-
-                {% if program.programpage %}
-                  </a>
+              {% endif %}
+                <div class="program-info">
+                  <h4 class="program-title" aria-describedby="program-{{ program.id }}-description">{{ program.programpage.title }}</h4>
+                  <div class="program-num-courses">
+                    {% blocktrans count counter=program.course_set.count %}
+                      {{ counter }} course
+                    {% plural %}
+                      {{ counter }} courses
+                    {% endblocktrans %}
+                  </div>
+                </div>
+                {% if program.programpage and program.programpage.thumbnail_image %}
+                  {% image program.programpage.thumbnail_image fill-690x530 as thumbnail_image %}
+                  <img src="{{ thumbnail_image.url }}" alt="course image for {{ program }}"
+                       >
+                {% else %}
+                  <img src="{% static 'images/course-thumbnail.png' %}"
+                       alt="course image for {{ program }}" class="program-default-image">
                 {% endif %}
-                <div class="program-thumbnail-description">
-                  <p>{{ program.description|default:"No description available for this program." }}</p>
+              {% if program.programpage %}
+                </a>
+              {% endif %}
+            </div>
+            <div class="program-description">
+              <p id="program-{{ program.id }}-description">{{ program.description|default:"No description available for this program." }}</p>
```

Comment:
The `<h4>` element for the program title has an [`aria-describedby`](http://rawgit.com/w3c/aria/master/aria/aria.html#aria-describedby) attribute that references this ID. It allows the screen reader to read the program description immediately after reading the program title.